### PR TITLE
Add goodmod.yaml for tracking module versions

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: goodmod.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - github.com/kubernetes-sigs/kube-storage-version-migrator
+    commit: bdca3bf7d454105b7853d0d60b385877486a9dc6
+
+  - paths:
+    - github.com/openshift/*
+    branch: master
+
+  - paths:
+    - github.com/jteeuwen/go-bindata
+    commit: a0ff2567cfb70903282db057e799fd826784d41d
```
[the remark this answers is not in the document]
Comment (sttts) [reply goodmod.yaml]: I thought a replace statement (plus required of course) with this sha is enough. The only information we lose with go.mod are non-semver tags (like those of kube) and branch names.

File: goodmod.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    - k8s.io/klog
+    - k8s.io/utils
+    tag: kubernetes-1.16.2
+
+  - paths:
+    - github.com/kubernetes-sigs/kube-storage-version-migrator
+    commit: bdca3bf7d454105b7853d0d60b385877486a9dc6
+
+  - paths:
+    - github.com/openshift/*
```
Comment (sttts) [inline goodmod.yaml]: I would rather like to see these written out, not with a wildcard.

File: goodmod.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfojtik) [reply goodmod.yaml]: i think only `required` is enough, we don't have to pin commits in goodmod.yaml unless we want to track why we pin to a particular commit. i updated the goodmod.yaml.

File: goodmod.yaml (hunk 2 above)
Comment (mfojtik) [reply goodmod.yaml]: fixed.

**Comment by mfojtik** [comment]
@sttts updated, PTAL

**Pull request closed without merging**